scripts/multiqc

```diff
@@ -478,11 +465,12 @@ plots_flat, plots_interactive, make_pdf, config_file, cl_config, verbose, quiet,
                     logger.warning("Deleting    : {}   (-f was specified)".format(os.path.relpath(config.data_dir)))
                     shutil.rmtree(config.data_dir)
                 else:
-                    logger.error("Output directory {} already exists.".format(config.data_dir))
-                    logger.info("Use -f or --force to overwrite existing reports")
-                    shutil.rmtree(tmp_dir)
-                    sys.exit(1)
-            os.makedirs(config.data_dir)
+                    data_num = 0
```

Comment:
I still think that it's possible to get numeric suffixes out of sync here. For example, if there is no `multiqc_report.html` but there is a `multiqc_data` directory, then we'll end up with `multiqc_report.html` and `multiqc_data_1`, no?

I think we'd be better off rolling these two checks into one. Something like:

```python
config.output_fn = os.path.join(config.output_dir, config.output_fn_name)
config.data_dir = os.path.join(config.output_dir, config.data_dir_name)
if os.path.exists(output_name) or os.path.exists(config.data_dir):
    # [..]
    report_num = 1
    while os.path.exists(output_name) or os.path.exists(config.data_dir):
       # [..]
```

Would need a bit more logic, for example making sure that stuff like `config.make_data_dir` still works.. Perhaps we could set `config.data_dir` to `None` in that case, and then `os.path.exists(str(config.data_dir))` should return `False` assuming that there isn't a file called `None` in the working directory..?